src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+
+    add(operand) {
+      calculator.result += operand;
+    },
+
+    subtract(operand) {
+      calculator.result -= operand;
+    },
+
+    divide(operand) {
+      calculator.result /= operand;
+    },
```

Comment:
what if there will be zero?)) we can't divide by zero (it was mentor's advice for me)

Reply:
I agree. Added check for 0.